internal/support/scheduler/schedule.go

```
@@ -506,8 +506,12 @@ func execute(
 			lc.Error("create new request occurs error : " + err.Error())
 		}
 
+		// Note this is V1 code that will be removed soon.
+		// Change is so code will compile, but not worried about error checking.
+		timeout, _ := time.ParseDuration(configuration.Service.RequestTimeout)
+
```

nice comment so we know why error is ignored!